### DISCUSSION TITLE
Way to add fully functional custom html block

### DISCUSSION THREAD

**Discussion by craeckor** [JulianPrieber/littlelink-custom#812] [Feature requests]
As the title already says it, it would be very nice to add a custom HTML-block to your page. This would create a lot of possibilities like add a (background) media-player or something other.

**Comment by JulianPrieber** [comment]
Something like this can be made using the latest beta.

**Reply by craeckor** [reply]
> Something like this can be made using the latest beta.

And where exactly is this latest beta 'cause I haven't found it.

**Reply by JulianPrieber** [reply]
If you are looking for how to update to the latest beta here is how to do it:
Toggle this option in your instance config and update
![grafik](https://github.com/user-attachments/assets/961846fe-d3cb-450a-b39d-496a452192c6)

**Reply by craeckor** [reply]
Ah, found it.
Thanks!

**Reply by craeckor** [reply]
> Something like this can be made using the latest beta.

um so I created a test-instace with the beta on it but I don't really see an option to achieve this

**Reply by JulianPrieber** [reply]
You'll have to make a plugin to do this

**Reply by JulianPrieber** [reply]
We're currently testing a modular block system
https://github.com/LinkStackOrg/LinkStack/tree/main/blocks

**Reply by craeckor** [reply]
Oh okay
